docs-ref-services/batch.md

```diff
@@ -59,4 +53,53 @@ new_pool.virtual_machine_configuration = vmc
 client.pool.add(new_pool)
 ```
 
-Explore more [sample Python code](https://azure.microsoft.com/resources/samples/?platform=python) you can use in your apps.
\ No newline at end of file
+## Management API
+Use the Azure Batch management libraries to create and delete batch accounts, read and regenerate batch account keys, and manage batch account storage.
+
+```bash
+pip install azure-mgmt-batch
+```
+
+### Example
+Create an Azure Batch account and configure a new application and Azure storage account for it.
+
+```python
+from azure.mgmt.batch import BatchManagementClient
+from azure.mgmt.resource import ResourceManagementClient
+from azure.mgmt.storage import StorageManagementClient
+
+LOCATION = 'eastus'
+GROUP_NAME ='batchresourcegroup'
+
+# Create Resource group
+print('Create Resource Group')
+resource_client.resource_groups.create_or_update(GROUP_NAME, {'location': LOCATION})
+
+# Create a storage account
+storage_params = azure.mgmt.storage.models.StorageAccountCreateParameters(
+    location=LOCATION,
+    account_type=azure.mgmt.storage.models.AccountType.standard_lrs
+)
```

Comment:
This way of creating a Storage account is not working anymore:
https://azure.microsoft.com/fr-fr/resources/samples/storage-python-manage/

```python
storage_async_operation = storage_client.storage_accounts.create(
    GROUP_NAME,
    STORAGE_ACCOUNT_NAME,
    StorageAccountCreateParameters(
        sku=Sku(SkuName.standard_ragrs),
        kind=Kind.storage,
        location='westus'
    )
)
storage_account = storage_async_operation.result()
```